src/explore-education-statistics-frontend/src/services/glossaryService.ts

```diff
@@ -1,20 +1,16 @@
 import { contentApi } from '@common/services/api';
-
-export interface GlossaryEntry {
-  title: string;
-  slug: string;
-  body: string;
-}
-
-export interface GlossaryCategory {
-  heading: string;
-  entries: GlossaryEntry[];
-}
+import {
+  GlossaryCategory,
+  GlossaryEntry,
+} from '@common/services/types/glossary';
 
 const glossaryService = {
   listGlossaryEntries(): Promise<GlossaryCategory[]> {
     return contentApi.get('/glossary', {});
   },
+  getGlossaryEntry(slug: string): Promise<GlossaryEntry> {
+    return contentApi.get(`glossary/${slug}`, {});
```

Comment:
Do you need the empty braces at the end there?

Reply:
Also removed it from all other calls (including in the admin glossaryService).